BackendBench/scripts/get_tests_stat.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+This is a helper script to analyze the test suite and provide statistics about the number of tests per operation.
+"""
+
+import click
+import statistics
+import traceback
+
+import torch
+from BackendBench.facto_suite import FactoTestSuite
+from BackendBench.suite import SmokeTestSuite
+from BackendBench.opinfo_suite import OpInfoTestSuite
+from BackendBench.torchbench_suite import DEFAULT_HUGGINGFACE_URL, TorchBenchTestSuite
+from BackendBench.scripts.pytorch_operators import extract_operator_name
+
+
+def analyze_test_suite(suite):
+    test_counts = {}
+    total_tests = 0
+    total_perf_tests = 0
+
+    print(f"Analyzing suite: {suite.name}")
+
+    for op_test in suite:
+        op_str = str(op_test.op)
+        op_name = extract_operator_name(op_str)
+
+        num_tests = 0
```

Comment:
nit: rename the num_tests -> num_correctness_tests and rename similar variables as well